### DISCUSSION TITLE
GPU fans spike from Idle

### DISCUSSION THREAD

**Discussion by IssacI19** [Rem0o/FanControl.Releases#2030] [Q&A]
When doing simple tasks, and sometimes practically idle, my GPU fans would spike up randomly and then stop spinning. (It's normal for my GPU fans to not be spinning under small browser tasks). Whats not normal is the random spike I've been getting. How can I fix that?

**Comment by Rem0o** [comment]
Have you configured your GPU with FanControl? Have you set a start%? What's your curve like, any hysteresis? Gotta give some info.

**Reply by IssacI19** [reply]
![image](https://github.com/Rem0o/FanControl.Releases/assets/137911543/421d9d6b-a18e-4d9b-a0cb-b7b1a4b79878)

This is a screenshot of my GPU Fan Control speed.

**Reply by Rem0o** [reply]
Could you also send a screenshot of the corresponding control cards make them expanded so we can see all parameters?

**Reply by Rem0o** [reply]
Also, do you have other software like AfterBurner with a fan curve set?

**Reply by IssacI19** [reply]
I think that it
![image](https://github.com/Rem0o/FanControl.Releases/assets/137911543/1b0aa21a-361c-49b5-91a1-510d479a746b)

**Comment by solalla1972** [comment]
Configura la curva de ventilación a partir del 49% para que salten los ventiladores. Por debajo de ese porcentaje, lo ventiladores estarán a 0 rpm.
En mi rtx 3060 funcionan a partir del 49%.
